docset/winserver2022-ps/smbshare/New-SmbShare.md

```diff
@@ -277,9 +304,12 @@ Accept wildcard characters: False
 ```
 
 ### -FullAccess
-Specifies which accounts are granted full permission to access the share.
-Use a comma-separated list to specify multiple accounts.
-An account may not be specified more than once in the *FullAccess*, *ChangeAccess*, or *ReadAccess* parameter lists, but may be specified once in the *FullAccess*, *ChangeAccess*, or *ReadAccess* parameter list and once in the *NoAccess* parameter list.
+
+Specifies which accounts are granted full permission to access the share. Use a comma-separated list
+to specify multiple accounts. An account may not be specified more than once in the **FullAccess**,
+**ChangeAccess**, or **ReadAccess** parameter lists, but may be specified once in the
+**FullAccess**, **ChangeAccess**, or **ReadAccess** parameter list and once in the **NoAccess**
+parameter list.
```

Comment:
Suggest striking the second sentence.
```suggestion
Specifies which accounts are granted full permission to access the share. An account may not be
specified more than once in the **FullAccess**, **ChangeAccess**, or **ReadAccess** parameter
lists, but may be specified once in the **FullAccess**, **ChangeAccess**, or **ReadAccess**
parameter list and once in the **NoAccess** parameter list.
```

Reply:
It is kind of like that. I've updated the text as below:

> Specifies which accounts are granted full permission to access the share. Use a comma-separated list
to specify multiple accounts. Each entry in the comma-separated list must be contained within single
or double quotes, for example `'CONTOSO\Finance Users','CONTOSO\HR Users'`.
> 
> An account may not be specified more than once in the **FullAccess**, **ChangeAccess**, or
**ReadAccess** parameter lists, but may be specified once in the **FullAccess**, **ChangeAccess**,
or **ReadAccess** parameter list and once in the **NoAccess** parameter list.